### PR TITLE
fix LDAP jigasi auth

### PR DESCRIPTION
**Pull request by soletan**
This merge request is conditionally adding configuration suggested in #500 to jigasi SIP configuration file when LDAP authentication has been selected in .env file.

### CODE REVIEW

**Comment by saghul** [comment]
I don't see how this is specific to LDAP based authentication.

@damencho In general, does it make sense to have this option set at all times?

**Comment by damencho** [comment]
Yep, this can be set all the time. It also optimizes some room discovery happening in some cases before joining if I recall correctly...

**Comment by soletan** [comment]
Well, even better ... I just don't know the underlying system and so I tried to satisfy #500 at most.

File: jigasi/rootfs/defaults/sip-communicator.properties [hunk 1]
```diff
@@ -142,3 +142,5 @@ org.jitsi.jigasi.transcription.SEND_TXT={{ .Env.JIGASI_TRANSCRIBER_SEND_TXT | de
 org.jitsi.jigasi.transcription.RECORD_AUDIO={{ .Env.JIGASI_TRANSCRIBER_RECORD_AUDIO | default "false"}}
 org.jitsi.jigasi.transcription.RECORD_AUDIO_FORMAT=wav
 {{end}}
+
+org.jitsi.jigasi.MUC_SERVICE_ADDRESS=muc.meet.jitsi
```
Comment (saghul) [inline jigasi/rootfs/defaults/sip-communicator.properties:146]: Please use the corresponding env variable here instead of hardcoding the default.

Comment (soletan) [reply jigasi/rootfs/defaults/sip-communicator.properties]: I don't know MUC, XMPP or anything related to the technical background of Jitsi service. All I know is the way docker files are created and scripts are used to establish configuration files here. Due to that I don't know if the variable picked in latest commit is the proper one for there are two variables in env.example using this value both looking like eligible candidates here.

Comment (saghul) [reply jigasi/rootfs/defaults/sip-communicator.properties]: No worries. This should be XMPP_MUC_DOMAIN, which defaults to `muc.meet.jitsi`

Comment (soletan) [reply jigasi/rootfs/defaults/sip-communicator.properties]: So it is.

**Comment by saghul** [comment]
Cheers!

**Pull request merged**